std_msgs/Header header

bool[] contact_states
geometry_msgs/PoseWithCovariance[] pose_vec
geometry_msgs/TwistWithCovariance[] twist_vec
